pkg/cluster/gvk/gvk.go

```diff
@@ -637,4 +643,10 @@ var (
 		Version: "v1beta1",
 		Kind:    "Kuadrant",
 	}
+
+	MLflow = schema.GroupVersionKind{
+		Group:   "mlflow.opendatahub.io",
+		Version: "v1alpha1",
+		Kind:    "MLflow",
```

Comment:
double check with Humair

Reply:
@HumairAK Has the MLFlow CRD been defined at this time?

Reply:
yes https://github.com/opendatahub-io/mlflow-operator/tree/main/api/v1